AutoGraph/WebSocketClient.swift

```diff
@@ -0,0 +1,126 @@
+import Foundation
+import Starscream
+
+public typealias WebSocketCompletionBlock = (Result<WebSocketClient.Events, Error>) -> Void
+
+open class WebSocketClient {
+    public typealias SerializedObject = Decodable
+
+    public enum Events {
+        case connected([String: String])
+        case disconnected(String, UInt16)
+        case data(SerializedObject)
+        case pong(Data?)
```

Comment:
@rex-remind101 Not sure if we should monitor these (ping, pong) or if they are needed. Thoughts?